### ISSUE TITLE
Prestashop SMS Login issue both IOS and android

### ISSUE THREAD

**Issue by avvvar** [inspireui/support#4637]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: 
Hello, I have tried to implement the SMS login but I am getting issues.

for IOS it was showing a notification error (if appdelegate swizzling is disabled ... etc), and now whenever i try to login with number the app will crash and give message (Lost connection to device.
Failed to send request: {"jsonrpc":"2.0","id":"88","method":"getMemoryUsage","params":{"isolateId":"isolates/2885774206765719"}} ) 

as for andorid , when i try to login using sms it gives the following error , ( this app is not authorized..) althought i completed the step carefully.

+ Product & version: Fluxstore Prestashop - Flutter E-commerce Full App 1.7.7
+ Flutter (or React Native) version:  flutter 1.20.2
+ Testing Device/Simulator:  Simulator I phone pro max
+ Screenshot issues (drag the file to attach here): 

<img width="397" alt="Screenshot 2020-08-17 at 8 41 33 PM" src="https://user-images.githubusercontent.com/39483126/90426700-2ced8280-e0ca-11ea-871b-bb5e6141ec11.png">

![Screenshot 2020-08-17 at 9 31 25 PM](https://user-images.githubusercontent.com/39483126/90431053-1dbe0300-e0d1-11ea-9570-79709cc09413.png)

**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com **

**_Important Note:_**
- Please help to ready the support policies before creating a new ticket: https://inspireui.com/support-policies/
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by minhcasi** [comment]
@avvvar - sorry the social and SMS login is not ready support for the Fluxstore Prestashop app - we will upgrade it later - thank you!

**Comment by avvvar** [comment]
Hi , 
okey understood,

**Comment by avvvar** [comment]
Hi, 
Understood that. 

with the new version, I got some new issues while ordering, please let me know how to solve them
1- when I select country, it doesn't display which country has been selected, it will remain on the "country" but when you order it will send the selected one .

2- while pressing the order, it will give the bellow error, and remain on the same page , but the order will be sent 
![Screenshot 2020-08-18 at 4 45 22 PM](https://user-images.githubusercontent.com/39483126/90522054-efddca80-e173-11ea-8885-0dbc6be20a1b.png)

please consider my settings for the address is as below 

firstname lastname
address1
city
Country:name
phone_mobile
phone

**Comment by avvvar** [comment]
Please support me with this ticket

**Comment by minhcasi** [comment]
@vumanhsn97 - please help to check the Checkout issue above - thank you!

**Comment by avvvar** [comment]
Hi @minhcasi @vumanhsn97 

 I have been waiting for your reply for a week, please support me

**Comment by avvvar** [comment]
hello?

**Comment by minhcasi** [comment]
@avvvar - we could not reproduce this issue on our demo - would you mind to share your current source to inspireui.team@gmail.com - we will help to investigate the error message on your checkout screens? 

We suspect it could cause by the data issue on your Rest API

**Comment by avvvar** [comment]
@minhcasi this could be due to my address settings which is set to the following 

firstname lastname
address1
city
Country:name
phone_mobile
phone

I have sent you the current lib.

please check the country selection as well, it is not functioning well.

**Comment by vumanhsn97** [comment]
@avvvar 
step 1: update countries code correctly in your database: https://tppr.me/7WPFv. Follow this page (column Alpha-2 code): https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
step 2: update default country and state config in lib/common/config/payments.dart https://tppr.me/8Cn9B
step 3: update this file in lib/services/prestashop.dart 
[prestashop.dart.zip](https://github.com/inspireui/support/files/5148304/prestashop.dart.zip)

**Comment by avvvar** [comment]
hi @vumanhsn97 

with current Prestashop.dart file the app is not even running, see the output

`Xcode's output:
↳
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/facebook_audience_network-0.7.0/ios/Classes/FacebookAudienceNetworkNativeAd.swift:59:21: warning: 'FBAdIconView' is deprecated: This class will be removed in a future release. Use FBMediaView instead.
    var adIconView: FBAdIconView!
                    ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/facebook_audience_network-0.7.0/ios/Classes/FacebookAudienceNetworkNativeAd.swift:164:13: warning: variable 'adType' was never mutated; consider changing to 'let' constant
        var adType: Int = self.params["ad_type"] as? Int ?? FANConstant.NATIVE_AD_TEMPLATE;
        ~~~ ^
        let
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/facebook_audience_network-0.7.0/ios/Classes/FacebookAudienceNetworkNativeAd.swift:324:27: warning: 'FBAdIconView' is deprecated: This class will be removed in a future release. Use FBMediaView instead.
        self.adIconView = FBAdIconView.init(frame: nativeAdLayout.adIconRect)
                          ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/facebook_audience_network-0.7.0/ios/Classes/FacebookAudienceNetworkNativeAd.swift:495:27: warning: 'FBAdIconView' is deprecated: This class will be removed in a future release. Use FBMediaView instead.
        self.adIconView = FBAdIconView.init(frame: nativeAdLayout.adIconRect)
                          ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/facebook_audience_network-0.7.0/ios/Classes/FacebookAudienceNetworkNativeAd.swift:596:38: warning: cast from 'Error' to unrelated type 'String' always fails
        let errorStr: String = error as? String ?? "";
                               ~~~~~ ^   ~~~~~~
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/facebook_audience_network-0.7.0/ios/Classes/FANPlugin.swift:26:26: warning: result of 'FANPluginFactory' initializer is unused
        FANPluginFactory.init(_channel: FANPluginChannel)
                         ^   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/facebook_audience_network-0.7.0/ios/Classes/FANPlugin.swift:30:53: warning: result of 'FacebookAudienceNetworkInterstitialAdPlugin' initializer is unused
        FacebookAudienceNetworkInterstitialAdPlugin.init(_channel: interstitialAdChannel)
                                                    ^   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.16/ios/Classes/FLTFirebaseMessagingPlugin.m:195:43: warning: 'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated, please use APNs for downstream message handling. [-Wdeprecated-declarations]
- (void)applicationReceivedRemoteMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                                          ^
In module 'FirebaseMessaging' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:93:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:219:1: note: 'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
__deprecated_msg(
^
In module 'UIKit' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/sys/cdefs.h:191:48: note: expanded from macro '__deprecated_msg'
        #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                      ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.16/ios/Classes/FLTFirebaseMessagingPlugin.m:307:24: warning: 'FIRMessagingRemoteMessage' is deprecated: FCM direct channel is deprecated, please use APNs for downstream message handling. [-Wdeprecated-declarations]
    didReceiveMessage:(FIRMessagingRemoteMessage *)remoteMessage {
                       ^
In module 'FirebaseMessaging' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:93:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:219:1: note: 'FIRMessagingRemoteMessage' has been explicitly marked deprecated here
__deprecated_msg(
^
In module 'UIKit' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/sys/cdefs.h:191:48: note: expanded from macro '__deprecated_msg'
        #define __deprecated_msg(_msg) __attribute__((__deprecated__(_msg)))
                                                      ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.16/ios/Classes/FLTFirebaseMessagingPlugin.m:141:30: warning: 'shouldEstablishDirectChannel' is deprecated: FCM direct channel is deprecated, please use APNs channel for downstream message delivery. [-Wdeprecated-declarations]
    [FIRMessaging messaging].shouldEstablishDirectChannel = true;
                             ^
In module 'FirebaseMessaging' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:93:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:283:56: note: 'shouldEstablishDirectChannel' has been explicitly marked deprecated here
@property(nonatomic) BOOL shouldEstablishDirectChannel DEPRECATED_MSG_ATTRIBUTE(
                                                       ^
In module 'UIKit' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/AvailabilityMacros.h:182:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
            #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                               ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.16/ios/Classes/FLTFirebaseMessagingPlugin.m:196:52: warning: 'appData' is deprecated [-Wdeprecated-declarations]
  [self didReceiveRemoteNotification:remoteMessage.appData];
                                                   ^
In module 'FirebaseMessaging' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:93:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:226:62: note: 'appData' has been explicitly marked deprecated here
@property(nonatomic, readonly, strong) NSDictionary *appData DEPRECATED_ATTRIBUTE;
                                                             ^
In module 'UIKit' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/AvailabilityMacros.h:180:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.16/ios/Classes/FLTFirebaseMessagingPlugin.m:306:1: warning: implementing deprecated method [-Wdeprecated-implementations]
- (void)messaging:(FIRMessaging *)messaging
^
In module 'FirebaseMessaging' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:93:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:253:1: note: method 'messaging:didReceiveMessage:' declared here
- (void)messaging:(FIRMessaging *)messaging
^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_messaging-6.0.16/ios/Classes/FLTFirebaseMessagingPlugin.m:308:63: warning: 'appData' is deprecated [-Wdeprecated-declarations]
  [_channel invokeMethod:@"onMessage" arguments:remoteMessage.appData];
                                                              ^
In module 'FirebaseMessaging' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:93:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/FirebaseMessaging/FirebaseMessaging/Sources/Public/FIRMessaging.h:226:62: note: 'appData' has been explicitly marked deprecated here
@property(nonatomic, readonly, strong) NSDictionary *appData DEPRECATED_ATTRIBUTE;
                                                             ^
In module 'UIKit' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Target Support Files/firebase_messaging/firebase_messaging-prefix.pch:2:
In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/AvailabilityMacros.h:180:60: note: expanded from macro 'DEPRECATED_ATTRIBUTE'
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
6 warnings generated.
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_analytics-5.0.16/ios/Classes/FLTFirebaseAnalyticsPlugin.m:56:19: warning: 'setScreenName:screenClass:' is deprecated: Use +[FIRAnalytics logEventWithName:kFIREventScreenView parameters:] instead. [-Wdeprecated-declarations]
    [FIRAnalytics setScreenName:screenName screenClass:screenClassOverride];
                  ^
In module 'FirebaseAnalytics' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:29:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/FirebaseAnalytics/Frameworks/FirebaseAnalytics.framework/Headers/FIRAnalytics.h:115:5: note: 'setScreenName:screenClass:' has been explicitly marked deprecated here
    DEPRECATED_MSG_ATTRIBUTE(
    ^
In module 'UIKit' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Target Support Files/firebase_analytics/firebase_analytics-prefix.pch:2:
In module 'Foundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
In module 'CoreFoundation' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
In module 'Darwin' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:16:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator13.6.sdk/usr/include/AvailabilityMacros.h:182:64: note: expanded from macro 'DEPRECATED_MSG_ATTRIBUTE'
            #define DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                               ^
1 warning generated.
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:80:37: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
- (void)rewardBasedVideoAd:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd
                                    ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:87:37: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
- (void)rewardBasedVideoAd:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd
                                    ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:95:49: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
- (void)rewardBasedVideoAdDidReceiveAd:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:100:44: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
- (void)rewardBasedVideoAdDidOpen:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                           ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:104:52: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
- (void)rewardBasedVideoAdDidStartPlaying:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                   ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:108:55: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
- (void)rewardBasedVideoAdDidCompletePlaying:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                      ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:112:45: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
- (void)rewardBasedVideoAdDidClose:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                            ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:117:57: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
- (void)rewardBasedVideoAdWillLeaveApplication:(nonnull GADRewardBasedVideoAd *)rewardBasedVideoAd {
                                                        ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:39:28: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
    [GADRewardBasedVideoAd sharedInstance].delegate = self;
                           ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:56:27: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
  [[GADRewardBasedVideoAd sharedInstance] loadRequest:[factory createRequest]
                          ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:61:27: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
  [[GADRewardBasedVideoAd sharedInstance]
                          ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:66:27: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
  [[GADRewardBasedVideoAd sharedInstance] setUserIdentifier:userIdentifier];
                          ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:70:27: warning: 'GADRewardBasedVideoAd' is deprecated: Use GADRewardedAd instead. Google AdMob publishers, follow instructions here: https://googlemobileadssdk.page.link/admob-ios-rewarded-migration. Google Ad Manager publishers, follow instructions here: https://googlemobileadssdk.page.link/admanager-ios-rewarded-migration. [-Wdeprecated-declarations]
  [[GADRewardBasedVideoAd sharedInstance] setCustomRewardString:customRewardString];
                          ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRewardBasedVideoAd.h:17:1: note: 'GADRewardBasedVideoAd' has been explicitly marked deprecated here
GAD_DEPRECATED_MSG_ATTRIBUTE(
^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.h:6:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRewardedVideoAdWrapper.m:75:66: warning: implicit conversion loses integer precision: 'FLTRewardedVideoAdStatus' to 'int' [-Wshorten-64-to-32]
      (NSString *)rewardedStatusToString[[NSNumber numberWithInt:_rewardedStatus]];
                                         ~                       ^~~~~~~~~~~~~~~
14 warnings generated.
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.m:71:13: warning: 'testDevices' is deprecated: Use GADMobileAds.sharedInstance.requestConfiguration.testDeviceIdentifiers. [-Wdeprecated-declarations]
    request.testDevices = testDevices;
            ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.h:5:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:83:71: note: 'testDevices' has been explicitly marked deprecated here
@property(nonatomic, copy, nullable) NSArray<NSString *> *testDevices GAD_DEPRECATED_MSG_ATTRIBUTE(
                                                                      ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.h:5:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.m:91:15: warning: 'birthday' is deprecated [-Wdeprecated-declarations]
      request.birthday =
              ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.h:5:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:90:55: note: 'birthday' has been explicitly marked deprecated here
@property(nonatomic, copy, nullable) NSDate *birthday GAD_DEPRECATED_ATTRIBUTE;
                                                      ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.h:5:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'
#define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))
                                                ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.m:106:19: warning: 'gender' is deprecated [-Wdeprecated-declarations]
          request.gender = genderValue;
                  ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.h:5:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:87:47: note: 'gender' has been explicitly marked deprecated here
@property(nonatomic, assign) GADGender gender GAD_DEPRECATED_ATTRIBUTE;
                                              ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.h:5:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'
#define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))
                                                ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.m:116:14: warning: 'tagForChildDirectedTreatment:' is deprecated:  use [GADMobileAds.sharedInstance.requestConfiguration tagForChildDirectedTreatment] [-Wdeprecated-declarations]
    [request tagForChildDirectedTreatment:childDirected.boolValue];
             ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.h:5:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADRequest.h:125:5: note: 'tagForChildDirectedTreatment:' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE(
    ^
In module 'GoogleMobileAds' imported from /Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTRequestFactory.h:5:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
4 warnings generated.
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTMobileAd.m:39:17: warning: 'configureWithApplicationID:' is deprecated: Use [GADMobileAds.sharedInstance startWithCompletionHandler:] [-Wdeprecated-declarations]
  [GADMobileAds configureWithApplicationID:appId];
                ^
In module 'GoogleMobileAds' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:188:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADMobileAds.h:85:5: note: 'configureWithApplicationID:' has been explicitly marked deprecated here
    GAD_DEPRECATED_MSG_ATTRIBUTE("Use [GADMobileAds.sharedInstance startWithCompletionHandler:]");
    ^
In module 'GoogleMobileAds' imported from /Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Headers/Public/Firebase/Firebase.h:188:
/Users/avvvar/Desktop/Apps/Lellia Flutter/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMobileAdsDefines.h:25:56: note: expanded from macro 'GAD_DEPRECATED_MSG_ATTRIBUTE'
#define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))
                                                       ^
/Users/avvvar/flutter/.pub-cache/hosted/pub.dartlang.org/firebase_admob-0.9.3+4/ios/Classes/FLTMobileAd.m:97:79: warning: implicit conversion loses integer precision: 'FLTMobileAdStatus' to 'int' [-Wshorten-64-to-32]
  NSString *statusString = (NSString *)statusToString[[NSNumber numberWithInt:_status]];
                                                      ~                       ^~~~~~~
2 warnings generated.
lib/services/prestashop.dart:501:31: Error: The method 'Prestashop.getPaymentMethods' doesn't have the named parameter 'address' of overridden method 'BaseServices.getPaymentMethods'.
  Future<List<PaymentMethod>> getPaymentMethods(
                              ^
lib/services/index.dart:64:31: Context: This is the overridden method ('getPaymentMethods').
  Future<List<PaymentMethod>> getPaymentMethods(
                              ^
lib/services/prestashop.dart:728:32: Error: The method 'Prestashop.getHomeCache' has more required arguments than those of overridden method 'BaseServices.getHomeCache'.
  Future<Map<String, dynamic>> getHomeCache(String lang) async {
                               ^
lib/services/index.dart:100:32: Context: This is the overridden method ('getHomeCache').
  Future<Map<String, dynamic>> getHomeCache();
                               ^

Command PhaseScriptExecution failed with a nonzero exit code
note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description

`

**Comment by avvvar** [comment]
can you give me some priority, please 
 @vumanhsn97

**Comment by vumanhsn97** [comment]
@avvvar 
Try to update this module: 
[customapi.zip](https://github.com/inspireui/support/files/5165745/customapi.zip)

**Comment by avvvar** [comment]
@vumanhsn97 
Thank you I will check it and let you know soon

**Comment by avvvar** [comment]
Hi @vumanhsn97 

Thank you the issue is solved, however, it is sending the note in the "your note" field, can you advise on that?

**Comment by minhcasi** [comment]
@avvvar - could you help to explain more the note issue via screenshot as we are not sure to get your idea? Thank you for your time!

**Comment by avvvar** [comment]
Hi @minhcasi ,

while completing the order, you have a field that allows the customer to write a note about his order, as shown in the below image. and it is not working, I don't know is it in my case only or it is a general issue 

<img width="392" alt="Screenshot 2020-09-08 at 12 40 40 AM" src="https://user-images.githubusercontent.com/39483126/92417689-20f65d00-f16c-11ea-9ba0-559606a4375d.png">

**Comment by vumanhsn97** [comment]
@avvvar 
Try with this step or waiting in next version:
- update this module: 
[customapi.zip](https://github.com/inspireui/support/files/5187298/customapi.zip)
- add this code in lib/services/prestashop.dart, before var response = ... line: https://tppr.me/bLPtl
if ((cartModel.notes??'').isNotEmpty) {
      params += '&notes=${cartModel.notes}';
    }

**Comment by avvvar** [comment]
Hi @vumanhsn97 ,
i tried to implement your settings, now i am receiving this error since then, i also changed every thing back but still i face the issue while placing the order

<img width="440" alt="Screenshot 2020-09-09 at 1 54 31 AM" src="https://user-images.githubusercontent.com/39483126/92536908-7baeb800-f243-11ea-9bfc-19655dffee54.png">

**Comment by vumanhsn97** [comment]
@avvvar 
It is working well. Please remove your app in your devices and build again
Thanks!

**Comment by avvvar** [comment]
@vumanhsn97 
well, I noticed that the user can proceed with the checkout without even registering or logging in, and that is when the abve issue happens.

how to disable checkout without logging in?

**Comment by vumanhsn97** [comment]
@avvvar 
In lib/common/config/payments.dart, set "GuestCheckout": false https://tppr.me/GJQc4

**Comment by avvvar** [comment]
@vumanhsn97 
thank you very much, I have a couple of other questions if you don't mind

- fluxBuilder is not connecting to my API, does it support Prestashop or not, i am receiving this error. 

![Screenshot 2020-09-11 at 5 33 06 AM](https://user-images.githubusercontent.com/39483126/92846340-a8ec9900-f3f0-11ea-9c9f-f7c4b1bf2119.png)

- how to manually change the homepage, which files to edit?


- also, I could not find how to change this loading logo 

<img width="235" alt="Screenshot 2020-09-11 at 5 30 44 AM" src="https://user-images.githubusercontent.com/39483126/92846350-abe78980-f3f0-11ea-91a8-7e0c23b8c252.png">


thank you

**Comment by vumanhsn97** [comment]
@avvvar 
Please follow this document: https://docs.inspireui.com/fluxstore/customization/
https://tppr.me/eWcC3
https://tppr.me/C83Yg

**Comment by avvvar** [comment]
Hi @vumanhsn97 
I am still facing the following issues...

- I changed the flr file and logo in the assets, but still seeing the below image while loading in search page 
<img width="235" alt="92846350-abe78980-f3f0-11ea-91a8-7e0c23b8c252" src="https://user-images.githubusercontent.com/39483126/93035227-1a248a00-f645-11ea-9aca-0c8a5b130859.png">

- the Story view is not working in my app, when I am implementing it, it will show as below ,
<img width="410" alt="Screenshot 2020-09-14 at 4 51 50 AM" src="https://user-images.githubusercontent.com/39483126/93035597-1b09eb80-f646-11ea-8347-ff948a0c02f9.png">

-  in case of setting the privacy page, when I set the pageId it is not fining it 


- is it possible to load the config_xx.json file from the internet ? how ?


thank you very much

**Comment by vumanhsn97** [comment]
@avvvar 
- story view: try with this template 
[config_theme1.json.zip](https://github.com/inspireui/support/files/5217569/config_theme1.json.zip)
- it is not support for prestashop. Try to replace with webview url
- upload config_xx.json to your site and replace link to your url in lib/common/config/general.dart https://tppr.me/RnKHU
- it is working well but not show in desktop(fluxbuilder preview)

**Comment by avvvar** [comment]
Dear @vumanhsn97 

- I have used your theme, and it is still showing it in the same way

<img width="397" alt="Screenshot 2020-09-15 at 1 23 04 PM" src="https://user-images.githubusercontent.com/39483126/93199558-37497d80-f757-11ea-9905-d4ad206e3a0d.png">

- as for layout in " const kAppConfig = 'lib/config/config_en.json' ", is it available only for the main language? , what about the others how to set them?

- can you please share with me the recent products view code for Prestashop?

**Comment by vumanhsn97** [comment]
@avvvar 
- story: may be it is not support for regular license
- multi language layout: you can upload same format url(config_xx.json). Ex: url/config_en.json, url/config_vi.json, ur/config_ar.json, ...
- recent view: {
      "name": "Recent View",
      "layout": "recentView"
    }
